frontend/src/sagas.js

```diff
@@ -800,11 +804,30 @@ export function* loginSaga({ user }) {
       isWarning: false
     });
   }
+
   function* onLoginError(error) {
-    yield put({
-      type: SHOW_LOGIN_ERROR,
-      show: true
-    });
+    if (error.response.status === 403) {
+      // user is disabled
+      yield put({
```

Comment:
I wouldn't expect that the disabled user error is shown below the password field.

Reply:
It is at the same position as "user password wrong" - error. Suggestions?

Reply:
done - all login errors have been moved to the snackbar